app/main/views/suppliers.py

```diff
@@ -64,11 +66,18 @@ def list_suppliers():
     else:
         suppliers = Supplier.query.order_by(Supplier.name, Supplier.supplier_id)
 
+    # Can search by either DUNS or Company Registration number but not both
     if duns_number:
         is_valid_string_or_400(duns_number)
         suppliers = suppliers.filter(
             Supplier.duns_number == duns_number
         )
+    elif company_registration_number:
+        is_valid_string_or_400(company_registration_number)
+        # TODO: enable search by other registration number
```

Comment:
This is a really minor point, but I feel this TODO is in the wrong place? If we were to add searching by other companies numbers, wouldn't it be as part of the if..elif block?

Reply:
I removed that in the end - we're not going to need to search by the other number.

Reply:
Okay, no worries